src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1643,4 +1643,20 @@
     <!-- Type is not significant for this node type; always null -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="always"/>
   </Node>
+  
+  <AbstractNode Name="BoundMethodBodyBase" Base="BoundNode">
+    <!-- Type is not significant for this node type; always null -->
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="always"/>
+  
+    <Field Name="BlockBody" Type="BoundBlock" Null="allow"/>
+    <Field Name="ExpressionBody" Type="BoundBlock" Null="allow"/>
+  </AbstractNode>
+
+  <Node Name="BoundNonConstructorMethodBody" Base="BoundMethodBodyBase">
+  </Node>
+
+  <Node Name="BoundConstructorMethodBody" Base="BoundMethodBodyBase">
```

Comment:
Do those new bound nodes need a custom `Display` implementation (see `Formatting.cs`)?

Reply:
> Do those new bound nodes need a custom Display implementation (see Formatting.cs)?

Display is defined only for expressions, these nodes are not expressions.

---
In reply to: [170112064](https://github.com/dotnet/roslyn/pull/24918#discussion_r170112064) [](ancestors = 170112064)